howto/install-appliance/landing.md

```diff
@@ -1,17 +1,18 @@
+The guides in this section describe how to install the Anbox Cloud Appliance on different cloud platforms. There is a difference between the full Anbox Cloud installation and the Anbox Cloud Appliance (see [Variants](https://discourse.ubuntu.com/t/anbox-cloud-overview/17802#variants)). This section focuses on the **Anbox Cloud Appliance**. For instructions on how to install **Anbox Cloud**, see [How to install Anbox Cloud](https://discourse.ubuntu.com/t/install-anbox-cloud/24336).
 
-[note type="information" status="Note"]There is a difference between the full Anbox Cloud installation and the Anbox Cloud Appliance (see [Variants](https://discourse.ubuntu.com/t/anbox-cloud-overview/17802#variants)). This section focuses on the **Anbox Cloud Appliance**. For instructions on how to install **Anbox Cloud**, see [How to install Anbox Cloud](https://discourse.ubuntu.com/t/install-anbox-cloud/24336).[/note]
+We strongly recommend that you follow the [Install the Anbox Cloud Appliance on a local machine](https://discourse.ubuntu.com/t/install-appliance/22681) tutorial before you install the appliance on a cloud platform. The tutorial guides you through installing the appliance on a local machine dedicated to Anbox Cloud and makes you familiar with the installation process and the general concepts of the Anbox Cloud Appliance.
 
-Make sure to check the [Requirements](https://discourse.ubuntu.com/t/installation-requirements/17734) before you start your installation.
+Always check the [Requirements](https://discourse.ubuntu.com/t/installation-requirements/17734) before you start your installation.
 
+## Supported cloud platforms
 The Anbox Cloud Appliance is currently available for the following cloud platforms:
 
 * [AWS](https://discourse.ubuntu.com/t/how-to-install-the-appliance-on-aws/29703)
 
 Other clouds are also supported, but the Anbox Cloud Appliance is not available from their application directories yet. Therefore, to install the appliance on such a cloud, install the Anbox Cloud Appliance snap on a cloud machine. See the following instructions:
 
-* For [Azure](https://azure.microsoft.com/): [How to install the appliance on Azure](https://discourse.ubuntu.com/t/how-to-install-the-appliance-on-azure/30824)
-* For [Google Cloud](https://cloud.google.com/) or [Oracle Cloud (OCI)](https://www.oracle.com/cloud/): [Install the Anbox Cloud Appliance on your local machine](https://discourse.ubuntu.com/t/install-appliance/22681)
+* For [Azure](https://azure.microsoft.com/) : [How to install the appliance on Azure](https://discourse.ubuntu.com/t/how-to-install-the-appliance-on-azure/30824)
+* For [Google Cloud](https://cloud.google.com/) : [How to install the appliance on Google Cloud](https://discourse.ubuntu.com/t/how-to-install-the-appliance-on-google-cloud/36254)
+* For [Oracle Cloud (OCI)](https://www.oracle.com/cloud/) : [Install the Anbox Cloud Appliance on a local machine](https://discourse.ubuntu.com/t/install-appliance/22681)
```

Comment:
The linked page is not really OCI specific but we used it as a pointer for users to begin with. However it does not call out any OCI specific. Shouldn't we drop it the OCI item with that?

Reply:
From the context, I'd suggest we keep it as it's part of the `Supported cloud platforms`

Reply:
Making it generic as discussed in the SU.